src/chrome/content/rules/RawGit.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	For RawGit.com
+
+		Works:
+
+			- $
+			- cdn
+
+		Doesn't Work:
+			- www (mismatch)
+-->
+<ruleset name="RawGit">
+  <target host="rawgit.com" />
+  <target host="*.rawgit.com" />
+
+  <rule from="^http://(?:(cdn\.)|www\.)?rawgit\.com/"
```

Comment:
This syntax took a little while to grok :)

Could we just use not match such as (?!www\.) ?